op-chain-ops/genesis/config.go

```diff
@@ -169,6 +169,89 @@ type DeployConfig struct {
 	FundDevAccounts bool `json:"fundDevAccounts"`
 }
 
+// Copy will deeply copy the DeployConfig
+func (d *DeployConfig) Copy() *DeployConfig {
```

Comment:
Q: Would it be reasonable and lower-maintence to implement `Copy` by just serialising to JSON and back? It's probably not the end of the world to have to list every field here but if we can do the copy automatically that would be nicer and it doesn't seem like something that needs to be hugely optimised.